app/account/services_views.py

```diff
@@ -0,0 +1,216 @@
+import typing as tp
+from functools import wraps
+
+import jwt
+
+from datetime import timedelta, datetime
+
+from account.schemas import RegisterUser
+from account.serializers import account_serializer
+
+from settings import (
+    SECRET_KEY,
+    ACCESS_TOKEN_EXPIRE_MINUTES,
+    ACCESS_TOKEN_JWT_SUBJECT,
+    TOKEN_TYPE,
+    TOKEN_ALGORITHM,
+    REFRESH_TOKEN_EXPIRE_MINUTES,
+    REFRESH_TOKEN_JWT_SUBJECT,
+    DEFAULT_ACCESS_TOKEN_EXPIRE_MINUTES,
+    DEFAULT_REFRESH_TOKEN_EXPIRE_MINUTES,
+)
+from utils import type_model_data
+
+
+def create_new_account(data: RegisterUser) -> tp.Dict[str, tp.Union[str, bool]]:
+    """
+    Create new account
+    Args:
+        data: register data
+    Return:
+        dict with account data
+    """
+
+    from main import db
```

Comment:
why it's here?

Reply:
db.commit, in next version i removed this